frontend/src/scenes/retention/RetentionLineGraph.tsx

```diff
@@ -55,43 +58,31 @@ export function RetentionLineGraph({
                     dashboardItemId
                         ? undefined
                         : (point) => {
-                              const { index } = point
```

Comment:
Super nit: feels a bit weird to have `points` inside of `point`. I didn't look much past the surface level here, but I wonder if `clickedCoordinate` or something else may be a better name. Ignore me if this is following patterns elsewhere/is a bigger refactor.

Reply:
Yeah, the payload types are a bit weird. I change dit slightly to match the other usages but could use an overall cleanup